truss/templates/server/model_wrapper.py

```diff
@@ -39,28 +100,39 @@ def load(self):
         if _signature_accepts_keyword_arg(model_class_signature, "secrets"):
             model_init_params["secrets"] = SecretsResolver.get_secrets(self._config)
         self._model = model_class(**model_init_params)
+
         if hasattr(self._model, "load"):
             self._model.load()
-        self.ready = True
 
-    def preprocess(self, request: Dict) -> Dict:
+    async def preprocess(
```

Comment:
why is this async? doesn't seem to be awaiting anything.

Reply:
The backing execution code handles both cases so you could make the case that having async allows you to have a model that uses async, but I think it just adds some confusion.  I'll remove in cleanup.